src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,43 @@
+import { useContext, useEffect } from 'react';
+import { Loader } from '../Loader';
+import { TableList } from '../TableList';
+import { TableContext } from '../../store/TableContextProvider';
+import { getPeople } from '../../api';
+
+export const PeopleTable = () => {
+  const { people, setPeople, isLoading, setIsLoading, isError, setIsError } =
+    useContext(TableContext);
```

Comment:
Create a hook so you don't need to pass context every time
`const usePeople = () => useContext(TableContext);`